app-emulation/aranym/aranym-1.0.2.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Atari Running on Any Machine"
+DESCRIPTION="
+ARAnyM (Atari Running on Any Machine) is a multiplatform virtual machine for \
+running Atari ST/TT/Falcon operating systems and applications.
+"
+HOMEPAGE="http://github.com/aranym/aranym"
+COPYRIGHT="2001-2018 ARAnyM developer team"
+LICENSE="GNU GPL v2"
+
+REVISION="1"
+gitCommit=69d0f59cc164baf3bbc51ccf8f82097aa858a296
+SOURCE_URI="https://github.com/aranym/aranym/archive/$gitCommit.tar.gz"
+SOURCE_FILENAME="aranym-$gitCommit.tar.gz"
+SOURCE_DIR="aranym-$gitCommit"
+PATCHES="aranym-$portVersion.patch"
+CHECKSUM_MD5="0689620ac9290e4848c1120d8101d40b"
```

Comment:
```diff
 LICENSE="GNU GPL v2"
-
 REVISION="1"
-gitCommit=69d0f59cc164baf3bbc51ccf8f82097aa858a296
-SOURCE_URI="https://github.com/aranym/aranym/archive/$gitCommit.tar.gz"
+SOURCE_URI="https://github.com/aranym/aranym/releases/tag/ARANYM_${portVersion//\./_/}.tar.gz"
+CHECKSUM_SHA256="c0935aee14cca55c08e75a94b77dcd69f326ae563ae371b22c7b9d8efd82e24e"
-SOURCE_FILENAME="aranym-$gitCommit.tar.gz"
-SOURCE_DIR="aranym-$gitCommit"
+SOURCE_DIR="aranym-ARANYM_${portVersion//\./_/}"
 PATCHES="aranym-$portVersion.patch"
-CHECKSUM_MD5="0689620ac9290e4848c1120d8101d40b"
-CHECKSUM_SHA256=7f4c1caa2dd474ddcf14d837199047aa546bcb87fe72a5d04459e7b909cac90f
```
Our wiki page for this is here: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering).

BTW, we like to have an empty line before `BUILD_REQUIRES`.

Reply:
that SOURCE_URI does not seem to work, when i run haikuporter, i get:

Downloading: https://github.com/aranym/aranym/releases/tag/ARANYM_1_/0_/2.tar.gz ...

(note the '/'s after the underscores)

Reply:
Oops, sorry, my mistake.
```diff
-SOURCE_URI="https://github.com/aranym/aranym/releases/tag/ARANYM_${portVersion//\./_/}.tar.gz"
+SOURCE_URI="https://github.com/aranym/aranym/releases/tag/ARANYM_${portVersion//\./_}.tar.gz"
```
(I just removed the slash before `}`.)